src/main/docs/guide/aop/validation.adoc

```diff
@@ -2,12 +2,6 @@ Validation advice is one of the most common advice types you are likely to want
 
 Validation advice is built on https://beanvalidation.org/2.0/spec/[Bean Validation JSR 380], a specification of the Java API for bean validation which ensures that the properties of a bean meet specific criteria, using `javax.validation` annotations such as `@NotNull`, `@Min`, and `@Max`.
```

Comment:
should update this link to bean validation 3.0 once the jakarta namespace switch happens which will happen soon https://github.com/micronaut-projects/micronaut-validation/pull/49

Reply:
Agree, we will do in a separate PR